### DISCUSSION TITLE
API enchantment: Expose NetworkManager value in SceneManager.

### DISCUSSION THREAD

**Discussion by ghost** [FirstGearGames/FishNet#297] [Feature Request]
Hey, I am writing a SceneProcessor for FishNet, and I'd like to do some stuff depending on if it's client or server.
But `_networkManager` field in SceneManager is private.

In all other networked components NM is a public field, so could you also do the same here?
Thanks in advance.

**Comment by FirstGearGames** [comment]
Approved.

**Comment by FirstGearGames** [comment]
Added in 3.6.1
